src/autocomplete.js

```diff
@@ -164,38 +179,43 @@ class Autocomplete extends Component {
   }
 
   getControllerStateAndHelpers() {
-    const {highlightedIndex, value, isOpen, selectedItem} = this.state
+    const {highlightedIndex, inputValue, isOpen, value} = this.state
     const {
-      toggleMenu,
+      getButtonProps,
+      getInputProps,
+      getItemProps,
+      getRootProps,
       openMenu,
       closeMenu,
-      clearSelection,
+      toggleMenu,
       selectItem,
       selectItemAtIndex,
       selectHighlightedItem,
       setHighlightedIndex,
-      getRootProps,
+      clearSelection,
+    } = this
+    return {
```

Comment:
Could we expose `getItemFromIndex` while we're at it? Ref: https://github.com/paypal/react-kadabra/issues/71